src/nft/components/collection/Card.tsx

```diff
@@ -0,0 +1,358 @@
+    [asset, hovered, href]
+  )
+
+  const assetRef = useRef<HTMLDivElement>(null)
+
+  useLayoutEffect(() => {
+    if (hovered && assetRef.current?.matches(':hover') === false) toggleHovered()
+  }, [hovered])
+
+  return (
+    <CardContext.Provider value={providerValue}>
+      <Box
+        as="a"
+        href={href ? href : baseHref(asset)}
+        position={'relative'}
+        ref={assetRef}
+        borderRadius={'20'}
+        className={styles.notSelectedCard}
+        draggable={false}
+        onMouseEnter={() => toggleHovered()}
+        onMouseLeave={() => toggleHovered()}
+        transition="250"
+      >
+        {children}
+      </Box>
+    </CardContext.Provider>
+  )
+}
+
+/* -------- CARD IMAGE -------- */
+const Image = () => {
+  const { hovered, asset } = useCardContext()
+  const [noContent, setNoContent] = useState(!asset.smallImageUrl && !asset.imageUrl)
+  const [loaded, setLoaded] = useState(false)
+
+  return (
+    <>
+      {!noContent ? (
+        <Box display="flex" overflow="hidden">
+          <Box
+            as={'img'}
+            alt={asset.name || asset.tokenId}
+            width="full"
+            style={{
+              aspectRatio: 'auto',
+              transition: 'transform 0.4s ease 0s',
+              background: loaded
+                ? 'none'
+                : `linear-gradient(270deg, ${themeVars.colors.medGray} 0%, ${themeVars.colors.lightGray} 100%)`,
+            }}
+            src={asset.imageUrl || asset.smallImageUrl}
+            objectFit={'contain'}
+            draggable={false}
+            onError={() => setNoContent(true)}
+            onLoad={() => {
+              setLoaded(true)
+            }}
+            className={clsx(hovered && styles.cardImageHover)}
+          />
+        </Box>
+      ) : (
+        <NoContentContainer />
+      )}
+    </>
+  )
+}
+
+/* -------- CARD DETAILS CONTAINER -------- */
+interface CardDetailsContainerProps {
+  children: ReactNode
+}
+
+const DetailsContainer = ({ children }: CardDetailsContainerProps) => {
+  return (
+    <Row
+      position="relative"
+      paddingX="12"
+      paddingTop="12"
+      justifyContent="space-between"
+      flexDirection="column"
+      transition="250"
+    >
+      {children}
+    </Row>
+  )
+}
+
+const InfoContainer = ({ children }: { children: ReactNode }) => {
+  return (
+    <Box overflow="hidden" width="full">
+      {children}
+    </Box>
+  )
+}
+
+const PrimaryRow = ({ children }: { children: ReactNode }) => <Row justifyContent="space-between">{children}</Row>
+
+const PrimaryDetails = ({ children }: { children: ReactNode }) => (
+  <Row overflow="hidden" whiteSpace="nowrap">
+    {children}
+  </Row>
+)
+
+const PrimaryInfo = ({ children }: { children: ReactNode }) => {
+  return (
+    <Box
+      overflow="hidden"
+      whiteSpace="nowrap"
+      textOverflow="ellipsis"
+      color="blackBlue"
+      fontWeight="medium"
+      fontSize="14"
+      style={{ lineHeight: '20px' }}
```

Comment:
shouldn't line height just be a prop on box?

Reply:
it should be but it isn't - we need to add that